README.md

```diff
@@ -19,16 +28,60 @@ use {
   requires = {
     'nvim-lua/plenary.nvim',
     'nvim-treesitter/nvim-treesitter'
-    },
-  config = function() -- optional
-    require("ruby_nvim").setup({
-      test_cmd = "ruby",  -- the default value
-      test_args = {},  -- the default value
-    })
-  end,
+  }
 }
 ```
 
-# Disclaimer
+Example with [vim-plug](https://github.com/junegunn/vim-plug):
+
+```vim
+Plug 'nvim-lua/plenary.nvim'
+Plug 'nvim-treesitter/nvim-treesitter'
+Plug 'vinibispo/ruby.nvim'
+```
+
+---
+
+# Setup
+
+## Default config
+
+Ruby.nvim comes with sane defaults so that you can get going without having to add much to your config
+
+```lua
+local ruby_nvim = require("ruby_nvim")
+
+ruby_nvim.setup({
+  test_cmd = "ruby" -- command that :RubyTest is gonna run,
+  test_args = {} -- args that :RubyTest is gonna run
+})
+```
+
+---
+
+# Commands
+
+## Run
+
+### :RubyRun
+
+Runs given file (current or another) in a floating window
```

Comment:
Why is this different from [the Vim doc](https://github.com/vinibispo/ruby.nvim/blob/debfa50ac82bff053c803f030c40f064e26f2a60/doc/help.txt#L54)?

Reply:
Because we have written wrong in vimdoc

Reply:
Got it! Since we expect an optional argument, I think we can have `:RubyRun [file]` instead of just `:RubyRun` ; )

Reply:
Ok, I'mma change it